site/app/views/submission/HomeworkView.php

```diff
@@ -50,11 +50,12 @@ public function showGradeable(Gradeable $gradeable, $graded_gradeable, int $disp
             $this->core->getOutput()->addInternalModuleJs('grader-submission.js');
         }
 
-        // Only show the late banner if the submission has a due date
+        // Only show the late banner and daylight savings banner if the submission has a due date
         // Instructors shouldn't see this banner if they're not on a team (they won't have proper information)
         if (LateDays::filterCanView($this->core, $gradeable) && !($is_admin && !$on_team && $is_team_assignment)) {
             $late_days = LateDays::fromUser($this->core, $this->core->getUser());
             $return .= $this->renderLateDayMessage($late_days, $gradeable, $graded_gradeable);
+            $return .= $this->renderDaylightSavingsMessage();
```

Comment:
Concatenating the output of multiple Twig files is poor form, and should be done by including Twig files within a single larger Twig file.  Given that this is already occurring extensively in this file, there are no action items here--I just wanted to note the issue.

Reply:
Good note.

Reply:
It is easier to do it from the late day message function, am going to change it back to what it was originally.